packages/sdk/src/types/Distributions.ts

```diff
@@ -139,16 +192,70 @@ export interface DistributionsPublicationsListResponse {
 	publications?: PublicationResponse[]
 }
 
-export interface DistributionsPublicationsCreateParams {
-	join_once?: boolean
+export type CreatePublicationViaGetRequest = DistributionsPublicationsCreateParams &
+	DistributionsPublicationsCreateQueryParams
+
+//5_req_create_publication
+export type DistributionsPublicationsCreateParams = (
```

Comment:
Before, `DistributionsPublicationsCreateParams` was the `Interface`; now, it's `Type`. In most cases, it is similar, but some differences can be non-backwards compatible (declaration merging, extends and intersection
). Describing it in the changelog or returning to the interface would be great.

Reply:
https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces